tests/Oryx.RuntimeImage.Tests/Node/NodeLegacyVersionsRuntimeImageTests.cs

```diff
@@ -68,6 +68,12 @@ public void NodeImage_Contains_VersionAndCommit_Information(string version)
         [InlineData("6.9", "6.9.5")]
         [InlineData("6.10", "6.10.3")]
         [InlineData("6.11", "6.11.5")]
+        [InlineData("8", NodeVersions.Node8Version)]
+        [InlineData("8.2", "8.2.1")]
+        [InlineData("8.11", "8.11.4")]
+        [InlineData("9.4", "9.4.0")]
+        [InlineData("10", NodeVersions.Node10Version)]
+        [InlineData("10.10", "10.10.0")]
```

Comment:
same here, look that they're no longer supported.

Reply:
actually the title of the pr is misleading .. we dont test extensively on these versions any more .. just a  basic level of end to end testing